source/modules/water/memory/ByteOrder.h

```diff
@@ -153,7 +153,7 @@ inline uint32 ByteOrder::swap (uint32 n) noexcept
 {
    #ifdef CARLA_OS_MAC
     return OSSwapInt32 (n);
-   #elif defined(CARLA_OS_WIN) || ! (defined (__arm__) || defined (__arm64__))
+   #elif defined(CARLA_OS_WIN) || ! (defined (__arm__) || defined (__aarch64__) || defined (__arm64__))

```

Comment:
nit-picking here, but please put the aarch64 check after the arm64 one.
so we check for arm, arm64 and then aarch64.
thanks

Reply:
OK. Done